products/identity-personalization/profile/feature-toggle-guidance.md

```diff
@@ -0,0 +1,37 @@
+# Auth Experience team feature toggle guide
+
+- [Auth Experience team feature toggle guide](#auth-experience-team-feature-toggle-guide)
+  - [Overview](#overview)
+  - [Team specific practices](#team-specific-practices)
+  - [Rough guidelines of how to implement a toggle](#rough-guidelines-of-how-to-implement-a-toggle)
+  - [Testing features in production](#testing-features-in-production)
+
+## Overview
+
+[Platform docs around feature toggles](https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/platform/tools/feature-toggles/)
+
+## Team-specific practices
+
+- Ideally, we use one toggle per feature.
+- A feature is an enhancement we would want to test and prove out before the end users can see/use it.
+- Always aim to deprecate a toggle after the feature is successfully deployed to production.
+- Namespace your toggles by naming the toggle using the `profile_` prefix. The toggle should be specific to your feature. For example: `profile_my_feature` or `profile_do_not_require_zip_code`.
+
+## Rough guidelines of how to implement a toggle
+
+- Add the toggle to the code base via the instructions in the platform docs.
+- Typically, FE and BE should use the same toggle unless there are unique requirements. Typically the fewer toggles, the better.
```

Comment:
Possibly add some context about using 2 toggles, and the guidance around rollout.

"If separate FE and BE feature toggles are required, then the BE toggle will need to be fully enabled during rollout to avoid any issues with 25/50/75% deployments" or something like that